substrate/frame/staking-async/src/tests/slashing.rs

```diff
@@ -683,18 +677,8 @@ fn slashing_nominators_by_span_max() {
 		let slash_1_amount = Perbill::from_percent(10) * nominated_value_11;
```

Comment:
probably better to use `add_slash_in_era_with_value` in the first invocation as well to make this clearer.

Reply:
Got rid of this helper setting implicit offence from everywhere.